com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -811,18 +832,14 @@ private static void mainInternal(String[] args) throws IOException, CompileExcep
         try {
             result = project.build(new ConsoleProgress(), commands);
         } catch(MultipleCompileException e) {
+            errors = parseMultipleException(e, verbose);
             ret = false;
-            errors.append("\n");
-            for (MultipleCompileException.Info info : e.issues)
-            {
-                errors.append(logExceptionToString(info.getSeverity(), info.getResource(), info.getLineNumber(), info.getMessage()) + "\n");
-            }
-
-            String msg = String.format("For the full log, see %s (or add -v)\n", e.getLogPath());
-            errors.append(msg);
-
-            if (verbose) {
-                errors.append("\nFull log: \n" + e.getRawLog() + "\n");
+        } catch(CompileExceptionError e) {
```

Comment:
Another fix broke this handling, and it may arrive as a CompileExceptionError whose cause is a MultipleCompileException.